index.html

```diff
@@ -191,9 +191,12 @@ <h2 class="font-bold uppercase text-gray-700 text-sm mt-6 mb-2">Organizátor</h2
           </div>
           <h2 class="text-2xl -mt-1">{{ name }}</h2>
 
-          <div class="hidden md:block -mt-1 mb-2">
-            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nihil blanditiis, nulla modi, natus doloribus temporibus. Natus tempore, tempora quam doloribus.
-          </div>
+          {{#info}}
+          <div class="hidden md:block -mt-1 mb-2">{{ info }}</div>
+          {{/info}}
+          {{^info}}
+          <div class="hidden md:block -mt-1 mb-2">Podrobnosti o tejto udalosti nie sú dostupné.</div>
```

Comment:
Ja by som nepisal nic, ani takyto textik, co si myslite?

Reply:
hej nad tým som dlho premýšlal, nakoniec som si povedal, že to môže vyzerať divne keď jeden event nič nebude mať (aby sa človek nesnažil to nejak rozklikávať alebo čo :D)